dcos/config.py

```diff
@@ -213,6 +213,26 @@ def get_config_schema(command):
     return subcommand.config_schema(executable, command)
 
 
+def get_property_description(name, value):
+    """
+    :param name: name of paramater
+    :type name: str
+    :param value: value to set to paramater `name`
+    :type param: str
+    :returns: description of name.value, or title if no description
+    :rtype: str
+    """
+
+    schema = get_config_schema(name)
+    properties = schema.get("properties")
+    config = properties.get(value)
+    if config is not None:
+        return config.get("description", "title")
```

Comment:
Right but `get('key', 'default value')` on a `dict` tries to find the key 'key'. If it can't find it, it returns 'title', no?